libraries/scripting/type-providers/src/kotlin/script/experimental/typeProviders/generatedCode/impl/generatedCodeFromFile.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package kotlin.script.experimental.typeProviders.generatedCode.impl
+
+import java.io.File
+import java.lang.StringBuilder
+import kotlin.script.experimental.host.toScriptSource
+import kotlin.script.experimental.typeProviders.generatedCode.GeneratedCode
+import kotlin.script.experimental.typeProviders.generatedCode.internal.visitor.GeneratedCodeVisitor
+import kotlin.script.experimental.typeProviders.generatedCode.internal.InternalGeneratedCode
+
+/**
+ * Include file as code generated.
+ */
+fun File.asGeneratedCode(): GeneratedCode =
+    GeneratedCodeInFile(this)
+
+/**
+ * Include jar file as generated code.
+ */
+fun File.asGeneratedJar(packageName: String): GeneratedCode =
```

Comment:
I'll write some tests. But this is in case the type provider has an already compiled jar they want to include.

I'm not sure if it will come up. I'm fine with removing it for now and seeing if it comes up

Reply:
Removed since it was a bit of an additional case. If at some point we need it, I'll add it back